test/e2e/persistent_volumes-disruptive.go

```diff
@@ -182,8 +182,6 @@ func testVolumeUnmountsFromDeletedPod(c clientset.Interface, f *framework.Framew
 	result, err = framework.SSH("mount| grep "+string(clientPod.UID), nodeIP, framework.TestContext.Provider)
 	Expect(err).NotTo(HaveOccurred())
 	Expect(result.Code).NotTo(BeZero())
-
```

Comment:
nit. L172 `framework.SSH("mount | grep "+string(clientPod.UID), nodeIP, framework.TestContext.Provider)`: 
would prefer to see `fmt.Sprintf()` used to construct the command string.